AlphaWallet/Extensions/RPCServer+Extensions.swift

```diff
@@ -144,3 +144,104 @@ extension RPCServer {
         }
     }
 }
+
+extension RPCServer: Comparable {
+    public static func < (lhs: RPCServer, rhs: RPCServer) -> Bool {
+        switch (lhs.isTestnet, rhs.isTestnet) {
+        case (true, false):
+            return false
+        case (false, true):
+            return true
+        default:
+            return lhs.displayOrderPriority < rhs.displayOrderPriority
+        }
+    }
+    public static func < (lhs: RPCServer, rhs: CustomRPC) -> Bool {
+        switch (lhs.isTestnet, rhs.isTestnet) {
```

Comment:
But `lhs` can be `RPCServer.custom`. Do we need to take that into account?

Reply:
The next two functions takes that into consideration.
`public static func < (lhs: CustomRPC, rhs: RPCServer) -> Bool`
`public static func < (lhs: CustomRPC, rhs: CustomRPC) -> Bool`

Reply:
I mean, in the test, we have:

 ```
 let c = RPCServer.goerli // chainId 5
 let d = CustomRPC.custom(chainId: 100)
 ```

 What if it's:

 ```
 let c = RPCServer.goerli // chainId 5
 let d = RPCServer.custom(CustomRPC.custom(chainId: 100))
 ```

 Maybe this is solved by one of the comparison functions. But seems it needs to be handled in here instead? —

```
public static func < (lhs: RPCServer, rhs: RPCServer) -> Bool {
```

Reply:
So where should this `RPCServer.custom` be in terms of precedence? Should it be treated the same as CustomRPC?

Reply:
Yes.

Reply:
Also, I don't think `RPCServer` ever gets compared to `CustomRPC` directly. Only indirectly vai `RPCServer.custom`  (in case you can remove some more code)